tests/kafkatest/services/connect.py

```diff
@@ -265,7 +265,8 @@ def node(self):
         return self.nodes[0]
 
     def start_cmd(self, node, connector_configs):
-        cmd = "( export KAFKA_LOG4J_OPTS=\"-Dlog4j.configuration=file:%s\"; " % self.LOG4J_CONFIG_FILE
+        cmd = "cd " + self.PERSISTENT_ROOT + " && "
```

Comment:
Unless I'm wrong, we move to this directory and that's where we execute all the rest of the commands (such as the echos in output files below). Just want to make sure this is what we want (which looks like it is)

Reply:
Though it does write to files inside the `PERSISTENT_ROOT`, it seems like (before this commit) it is executing from the user home directory `~/`. Because Connect seems to start looking for plugins from wherever the command was executed, seems like a `cd ...` beforehand is the simple fix.